docs/content/_index.adoc

```diff
@@ -6,9 +6,51 @@ draft: false
 
 {{< seo
     title="Verify access all the time and everywhere with Heimdall"
-    link="/docs/getting_started/concepts.adoc"
-    link_title="Get Started"
 >}}
-Adopt Zero Trust architecture by authenticating, enriching and authorizing incoming HTTP requests at any level. No need to learn new protocols, authentication and authorization frameworks. You can still use them.
+Heimdall is a cloud native Identity Aware Proxy and Access Control Decision service, which can help you adopting Zero Trust architecture by authenticating, enriching and authorizing incoming HTTP requests at any level. No need to learn new protocols, authentication and authorization frameworks.
+<br><br>
+<a class="btn btn-primary btn-lg" type="button" href="link:{{< relref "/docs/getting_started/concepts.adoc" >}}">Get Started</a>
 {{< /seo >}}
 
+{{% seo title="Easy to integrate" %}}
+
+Use technics you are already familiar with
+
+[source, yaml]
+----
+apiVersion: heimdall.dadrus.github.com/v1alpha2
+kind: RuleSet
+metadata:
+  name: My awesome service
+spec:
+  rules:
+    - id: my_api_rule
+      match:
+        url: http://127.0.0.1:9090/api/<**>
+      execute:
+        - authenticator: keycloak
+        - authorizer: opa
+----
+
+The decision process can be controlled by each and every upstream service individually via rules, which heimdall can load from different sources, like e.g. from a `RuleSet` kubernetes resource.
+
+
+{{% /seo %}}
+
+{{% seo title="Reduces the cognitive load of your team" %}}
+
+By outsourcing authentication and authorization decisions to heimdall you can reduce the complexity of your code base, free resources and reduce the cognitive load of your team pretty much.
```

Comment:
Not sure if "pretty much" is correct here, sounds a bit awkward. I'd either replace it with something like "substantially" or remove it.

Reply:
Yeah, you're right. Changed that based on your suggestion.

Reply:
Reasoning for adding that: I want to highlight that systems like heimdall can help reducing complexity. Actually, reducing is a wrong word. The complexity is there, but you can shift it, making it less problematic for the own code. Or even not having to deal with it at all. This already starts with the gained ability to decide how the service gets informed about the subject, and alike. All of that can make the implementation of the upstream service simpler and also much easier to test.

Reply:
Definitely agree with the content and I'd argue that if you're using a microservice architecture you are actually actively reducing complexity by centralizing it in one place.